contracts/interfaces/ILimitOrder.sol

```diff
@@ -57,9 +92,12 @@ interface ILimitOrder is IStrategyBase {
         bytes takerSig;
     }
 
-    /**
-     * Fill limit order by trader
-     */
+    /// @notice Fill an order by a trader
+    /// @notice Called by user proxy only
```

Comment:
nit:
```suggestion
    /// @notice Only user proxy can call
```

Reply:
Adjusted in 4fed1e9c109a6478d2a92f53649d77877b9f9668